boards/samr30-xpro/include/periph_conf.h

```diff
@@ -147,6 +147,7 @@ static const i2c_conf_t i2c_config[] = {
  */
 #define RTT_FREQUENCY                           (32768U)
 #define RTT_MAX_VALUE                           (0xffffffffU)
+#define RTT_MIN_VALUE                           (8U)
```

Comment:
would it make sense to add a note how this number was determined?

(8 seems quite a lot, that's 250us, right?)

Reply:
> (8 seems quite a lot, that's 250us, right?)

Yes, I was surprised, as well. I guess the synchronization between the different clock domains requires so much time.

Reply:
Done.